coresimd/x86_64/bmi.rs

```diff
@@ -14,10 +14,13 @@ use stdsimd_test::assert_instr;
 
```

Comment:
OCD: I just realized that this file is still called bmi.rs, but this and the one in x86 should be renamed to bmi1.rs to match the target feature name.